Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -145,6 +145,11 @@ public partial class EOSManager : MonoBehaviour, IEOSCoroutineOwner
         //cached log levels for retrieving later
         private static Dictionary<LogCategory, LogLevel> logLevels;
 
+        /// <summary>
+        /// If provided, this configuration file will be used when the Platform Interface is created.
+        /// </summary>
+        public EOSConfig OverridingEOSConfigForPlatformCreate { get; set; } = null;
```

Comment:
Since there is parallel work being done regarding fixing the override functionality of the configuration values, I think that this change is at odds with changes that are already in our development pipeline.

However, if this is the _only_ way to make the `feature/timeout` branch function properly, that would be understandable - but even if that's the case I think the function that accomplishes making it function properly should be done with a more narrowly described function.

Reply:
I'm comfortable waiting until the development pipeline is along and committed to the point where I can use that override. Do you have any estimate on how much work needs to be done until that is available?